connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/processors/it/ITUserRequestPreparer.java

```diff
@@ -0,0 +1,54 @@
+package com.redhat.cloud.notifications.connector.email.processors.it;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.cloud.notifications.connector.email.constants.ExchangeProperty;
+import com.redhat.cloud.notifications.connector.email.model.settings.RecipientSettings;
+import com.redhat.cloud.notifications.connector.email.processors.it.pojo.request.ITUserRequest;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.component.http.HttpMethods;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class ITUserRequestPreparer implements Processor {
```

Comment:
The call to the IT Users Service relies on a key store (see `*IT_S2S*` properties in the `engine` ClowdApp template). This looks to be missing in the connector.

Reply:
All right. I've created the parameters and also a private function in the `EmailRouteBuilder#setUpITEndpoint` which creates the IT endpoint with the provided Keystore and whatnot. However that might throw an exception, and I had to modify the `connector-common` package for it. I am starting to wonder if we should simply implement the BOP integration and go along with it...

Reply:
That would be simpler indeed, but last time we discussed that topic with the A&M team, I understood BOP wasn't ready yet for that.

Reply:
All right, we'll keep going then